Tools/WebKitTestRunner/wpe/EventSenderProxyClientWPE.cpp

```diff
@@ -303,6 +303,114 @@ void EventSenderProxyClientWPE::keyDown(WKStringRef keyRef, double time, WKEvent
     wpe_event_unref(event);
 }
 
+#if ENABLE(TOUCH_EVENTS)
+void EventSenderProxyClientWPE::addTouchPoint(int x, int y, double)
+{
+    uint32_t id = 0;
+    for (const auto& point : m_touchPoints) {
+        if (point.id == id)
+            id++;
+    }
```

Comment:
Can this fail if we remove previously added points from the array and we try to add a new points?

Reply:
No, it's ok to reuse ids.